GitUI/GitUICommands.cs

```diff
@@ -1471,6 +1471,8 @@ private bool RunCommandBasedOnArgument(IReadOnlyList<string> args, IReadOnlyDict
                     return StartViewPatchDialog(args.Count == 3 ? args[2] : "");
                 case "uninstall":
                     return UninstallEditor();
+                case ".":
```

Comment:
Should this be handled in default:?
A specific case for handling paths.

Reply:
Sure. Thanks for the great suggestion.
I have added that.

Reply:
The `default:` case is dealing with clone URLs.
Without a comment, the added code will be hard to understand next time we will read it - basically, i.e. not only if not handled using `case ".":`.

Reply:
I have added the comment as discussed. The changes will be reflected in the next push.